dcmio: dcmanonymizer: adding anonimyzer pipeline. With pilots

dgoyard opened the pull request:
dcmio: dcmanonymizer: adding anonimyzer pipeline. With pilots
dcmio: dcmreader: adding dicom reader functions (utils)

Commit:
dcmio: dcmanonymizer: adding anonimyzer pipeline. With pilots
dcmio: dcmreader: adding dicom reader functions (utils)

Commit:
dcmreader: improve values extraction from dicom using a recursive walker and callback system to browse enhanced storage format

AGrigis commented on dcmio/dcmreader/dcmreader.py:
```diff
@@ -0,0 +1,189 @@
+    return -1
+
+
+def get_SOP_storage_type(path_to_dicom):
```
sop

AGrigis commented on dcmio/dcmreader/dcmreader.py:
```diff
@@ -0,0 +1,189 @@
+    return False
+
+
+def get_Raw_Data_Run_Number(path_to_dicom):
```
raw_data_run_number

AGrigis commented:
-> remove *.pyc and *.py~ compiled/temp files.
-> add .gitignore.
->uniformize docstring and add 'Parameters' and 'Returns' descriptions.

Commit:
Delete __init__.pyc

Commit:
Delete __init__.pyc

Commit:
Delete anonymizer.pyc

Commit:
Delete anonymizer.py~

Commit:
Delete dicom_anonymizer.xml~

Commit:
Delete pilot.py~

Commit:
Delete __init__.pyc

Commit:
Delete dcm_to_nii.xml~

Commit:
Delete dicom_utils.pyc

Commit:
Delete pilots.pyc

Commit:
Delete pilots.py~

Commit:
Delete dcmreader.py~

Commit:
-> remove *.pyc and *.py~ compiled/temp files.
-> add .gitignore.
->uniformize docstring and add 'Parameters' and 'Returns' descriptions.

Commit:
Merge branch 'master' of https://github.com/dgoyard/caps-dcmio

dgoyard commented:
-> remove *.pyc and *.py~ compiled/temp files.                                                        OK
-> add .gitignore.                                                                                                       OK
->uniformize docstring and add 'Parameters' and 'Returns' descriptions.                 OK

Commit:
dcmreader.py  SOP -> sop

Commit:
dcmio: dcmanonymizer/anonymizer.py: adding header
dcmreader/dcmreader.py: adding header

Commit:
Merge branch 'master' of https://github.com/dgoyard/caps-dcmio
